ForceFeedback/ForceFeedback.Rules/MethodTooLongTextAdornment.cs

```diff
@@ -294,57 +309,16 @@ private Rect CalculateBounds(SyntaxNode syntaxNode, SnapshotSpan snapshotSpan)
             if (snapshotSpan == null)
                 throw new ArgumentNullException(nameof(snapshotSpan));
 
-            var left = CalculateLeftPosition(ref snapshotSpan);
-
-            if (left < 0)
-                return Rect.Empty;
-
-            var geometry = _view.TextViewLines.GetMarkerGeometry(snapshotSpan, true, new Thickness(0));
+            var geometry = _view.TextViewLines.GetMarkerGeometry(snapshotSpan, false, new Thickness(0));
```

Comment:
This change here disables clipping to the viewport when calculating the marker geometry. This is sufficient to fix bug #45. We shouldn't extend the method background to the full viewport width.